lib/slimmer/component_resolver.rb

```diff
@@ -56,7 +56,10 @@ def fetch(template_url)
     end
 
     def template_url(template_path)
-      [static_host, "templates", "#{template_path}.raw.html.erb"].join('/')
+      # accept full or partial component template_path from application
```

Comment:
I'm not sure this comment is necessary, since the detail in the commit that modified is good.